SwiftWisdom/Core/Foundation/Not.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Not.swift
+//  bmap
+//
+//  Created by Logan Wright on 12/11/15.
+//  Copyright © 2015 Intrepid. All rights reserved.
+//
+
+/**
+ Inverts a given boolean return closure
+ 
+ Good for things like `array.filter(not(matchesThing))`
+ */
+public func not<T>(original: T -> Bool) -> T -> Bool {
```

Comment:
Global functions are namespaced, so if two modules have declare `not`, we would have to specify like: `IntrepidSwiftWisdom.not()` or `SomeOtherModule.not(`